yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/SimpleProfileTest.java

```diff
@@ -36,8 +173,8 @@ public void is_shouldReturnBooleanValueForExistingKey() {
 
     @Test
     public void is_shouldReturnDefaultBooleanForNonExistingKey() {
-        Map<String, Object> map = Collections.emptyMap();
-        Profile profile = new SimpleProfile(map);
+        List<Attribute> list = Collections.emptyList();
```

Comment:
you don't need the list obj in here, you can directly call the `Collection.emptyList()` on SimpleProfile creation.

Reply:
I cannot as the type variable would be lost and the constructor would not accept it.